Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -312,7 +312,33 @@
 #if ENABLE(UNIFIED_TEXT_REPLACEMENT)
 void WebPage::getTextIndicatorForID(const WTF::UUID& uuid, CompletionHandler<void(std::optional<WebCore::TextIndicatorData>&&)>&& completionHandler)
 {
-    m_unifiedTextReplacementController->getTextIndicatorForID(uuid, WTFMove(completionHandler));
+    RefPtr range = m_unifiedTextReplacementController->rangeForUUID(uuid);
```

Comment:
Just to double check — does this return `nullptr` in the event `uuid` is not tin the map? I wonder if it's just returning `Ref(HashTableEmptyValueType)`, and then implicitly converting that into a `RefPtr`